src/main/telemetry/crsf.c

```diff
@@ -557,6 +629,29 @@ void crsfProcessDisplayPortCmd(uint8_t *frameStart)
 
 #endif
 
+#if defined(USE_CRSF_V3)
+void crsfProcessCommand(uint8_t *frameStart) {
+    uint8_t cmd = *frameStart;
+    uint8_t subCmd = frameStart[1];
+    switch (cmd) {
+        case CRSF_COMMAND_SUBCMD_GENERAL:
```

Comment:
Coding style in Betaflight is to have the `case` statements at the same indentation level as the `switch` statement.